lte/gateway/configs/service_registry.yml

```diff
@@ -110,3 +110,6 @@ services:
   kernsnoopd:
     ip_address: 127.0.0.1
     port: 50084
+  s8_service:
+    ip_address: 127.0.0.1
+    port: 50073
```

Comment:
why do you use the same port as SPGW service?

Reply:
As part of grpc initialization, grpc server is created with specific ip and port number. And there are multiple services like s1ap, sgs, ha etc are getting registred to this server. Similarly I have registered s8 service to existing server with same ip and port number.
Here is the link for initializing grpc server.
https://github.com/magma/magma/blob/master/lte/gateway/c/core/oai/tasks/grpc_service/grpc_service.cpp#L62

Reply:
ah indeed, you are reusing the grpc server task. then can you please place this together with 50073 services?